src/app/components/login/login.component.spec.ts

```diff
@@ -27,6 +27,11 @@ describe('LoginComponent', () => {
     expect(component).toBeTruthy()
   })
 
+  it('should validate email format', () => {
```

Comment:
This does not validate email format

Reply:
Okay thank you will do the necessary changes and push the code again

Reply:
Please changes has been done  and pushed to branch.
Please review and give feedback @Bilal-Abubakari